kwamp-client-core/src/main/kotlin/com/laurencegarmstrong/kwamp/client/core/Client.kt

```diff
@@ -153,6 +157,11 @@ class ClientImpl(
         topicPattern: UriPattern,
         eventHandler: EventHandler
     ) = subscriber.subscribe(topicPattern, eventHandler)
+
+
+    fun getID(): Long? {
+        return sessionId
+    }
```

Comment:
To keep this consistent with other simple functions, let's just do something like this?
```suggestion
    fun getSessionId() = sessionId
```
I think we should explicitly mention that it's the **session** ID in the function name too.
Also just for consistency, I think we can have the lowercase "d" in `Id` just to align with the variable name.